packages/server/test/course.integration.ts

```diff
@@ -298,11 +298,8 @@ describe('Course Integration', () => {
         staffList: [],
       });
 
-      const UCM = UserCourseModel.findOne({
-        where: { userId: ta.id, courseId: queue.courseId },
-      });
       const events = await EventModel.find({
-        where: { user: UCM },
+        where: { userId: ta.id },
```

Comment:
I'm fairly sure this test was originally incorrectly written - it should be finding Events with the given User, not the given UserCourseModel. It started failing as written on this branch - maybe TypeORM changed how it does joins?

Making this same switch on master caused the test to fail as well, because the course controller is actually logging a "checkout" event even if the TA wasn't originally in the queue. I've updated the course controller to not log an event.